public/sw.js

```diff
@@ -23,15 +23,27 @@ const fetchEventHandler = async event => {
     if (req.headers.has('accept')) {
       supportsWebp = req.headers.get('accept').includes('webp');
     }
+    // detect the ios version and remove webp from the request
+    // if supports webp is false in request header then don't use it?
+    // If we support WebP don't remove
+    // if it doesn't support, remove
+    // look at the headers Accept image/webp,*/
+    // downgrade/remove if not there
 
-    // If we support WebP
-    if (supportsWebp && !/\/amz\/worldservice\/.*/.test(event.request.url)) {
+    if (!supportsWebp) {
       event.respondWith(
-        fetch(`${req.url}.webp`, {
+        fetch(`${req.url}`.slice(0, -5), {
```

Comment:
Can we be really prescriptive here, and explicitly remove `.webp` from the URL. We'll probably need to assign it to another variable:

```suggestion
        let imageUrlWithoutWebp = req.url.replace('.webp', '');
        fetch(imageUrlWithoutWebp, {
```